styles.css

```diff
@@ -229,12 +229,130 @@ button:active {
   transition: 0.3s;
 }
 
+.grid-item2 {
+  display: flex;
+  align-self: flex-start;
+  justify-self: flex-start;
+}
+
+.grid-item2 ul {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.grid-item2 li {
+  font-size: 0.8em;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  margin-left: 5px;
+  margin-right: 5px;
+  margin-top: 30px;
+  padding: 0;
+  transition: 0.3s;
+}
+
 li:hover {
   color: #ebf0ee;
   background-color: #60c095;
   scale: 1.2;
 }
 
+.popup-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.8);
+  z-index: 999;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-content {
+  position: relative;
+  width: 80%;
+  max-width: 600px;
+  margin: 10% auto;
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 8px;
+}
+
+.close-button {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  font-size: 20px;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.name-container {
+  display: flex;
+  align-items: center;
+}
+
+.project-name {
+  margin-left: 0;
+}
+
+.button-group {
+  margin-left: auto;
+  display: flex;
+  gap: 10px;
+}
+
+.project-button {
+  display: inline-block;
+  padding: 10px 20px;
+  background-color: #36b37e;
+  margin-top: -5%;
+  color: #fff;
+  border: none;
+  border-radius: 4px;
+  font-size: 16px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+.project-button:hover {
+  background-color: #0056b3;
+}
+
+.project-button:focus {
+  outline: none;
+}
+
+.project-description {
+  display: flex;
+  text-align: left;
+  max-height: 200px;
+  overflow-y: auto;
+  margin-top: 20px;
+}
+
+.image-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  height: 200px;
+  overflow: hidden;
+}
+
+.featured-image {
+  width: auto;
+  height: auto;
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain;
+}
```

Comment:
- [ ] Great job on the implementation so far! 👍🏿 However, upon closer inspection, I noticed that the pop-up images don't fully cover the width of the pop-up on both desktop and mobile view screens. I kindly suggest adjusting the images to fill up the entire width of the pop-up, following the design in Figma.